tests/contrib/mcp/conftest.py

```diff
@@ -127,7 +127,6 @@ async def run_test():
             from mcp.shared.memory import create_connected_server_and_client_session
 
             async with create_connected_server_and_client_session(mcp_server._mcp_server) as client:
-                await client.initialize()
```

Comment:
this happens within `create_connected_server_and_client_session` (same below) - this avoids creating two spans